tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -437,10 +510,17 @@ public void SetMultipleIndexer(int index, string sIndex, int value)
 			{
 				this[index, sIndex] = value;
 			}
+
+			public int GetMultipleIndexer(int index, string sIndex)
+			{
+				return this[index, sIndex];
+			}
```

Comment:
This doesn't appear to be related with the addition of `SetupSet` and `VerifyGet`, and in either case, it doesn't appear to get used anywhere. Please remove.

Reply:
done